core/types/l2trace.go

```diff
@@ -21,9 +21,11 @@ var (
 	}
 )
 
-// BlockResult contains block execution traces and results required for rollers.
-type BlockResult struct {
-	BlockTrace       *BlockTrace        `json:"blockTrace"`
+// BlockTrace contains block execution traces and results required for rollers.
+type BlockTrace struct {
+	Coinbase         *AccountWrapper    `json:"coinbase"`
```

Comment:
One quick question. Why do we add an extra `Coinbase` here given the `Header` already contains the `Coinbase`?